impeller/shader_archive/multi_arch_shader_archive.fbs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+namespace impeller.fb;
+
+enum RenderingBackend:byte {
```

Comment:
Should we include more information here? Right now there are some implicit version constraints in these formats more or less. We could make this explicitly kOpenGLESv100 or add additional version metadata.

You could imagine in the future that we might support targeting higher GLSL versions for desktop or webgl, and the engine will need to know the right version information to consume it. I'm not sure if we'd ever ship multiple GLSL versions of the same shader in the same file.

Reply:
All of this is internal implementation detail. So I suspect we can tinker on this all we want.

In the future, we could even concatenate multiple shader libraries for the same backend instead of loading on of each.

Reply:
Sgtm